Tests/AK/TestSlugify.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2023, Gurkirat Singh <tbhaxor@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/Slugify.h>
+#include <LibTest/TestCase.h>
+
+TEST_CASE(ignore_unicode_characters)
+{
+    EXPECT_EQ(MUST(slugify(MUST(String::from_utf8("Hello World!🎉"sv)))), MUST(String::from_utf8("hello-world"sv)));
```

Comment:
You can use `""_string` for all of these tests. Instead of:
```c++
MUST(String::from_utf8("Hello"sv))
```
it'd be:
```c++
"Hello"_string
```

Reply:
Oh yeah. I forgot _string literal exists. It is cleaner. I am using it now. Thanks for checking.